### PR TITLE
Create an exercise for ES6 and Babel (Connects to #118)

### PR DESCRIPTION
**Pull request by ggprod**

### CODE REVIEW

**Comment by ggprod** [comment]
@rpdasilva @barretodavid hi guys, are you ok with this PR?

File: exercises/es6/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# ES6 and Babel Review Exercise
+
+An exercise to create a little node ES6 app, compiled with Babel
+
+![](./es6-exercise.gif)
+
+## Step 1
+
+Create a new folder called `es6` inside of the `training` folder.  Run `npm init` in the `es6` folder to initialize a new npm package.
```
Comment (rpdasilva) [inline exercises/es6/README.md]: `npm init -y` maybe to save a few steps

Comment (ggprod) [inline exercises/es6/README.md]: will update

File: exercises/es6/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## Step 2
+
+Install the Babel CLI in the `es6` folder with the command `npm i -D babel-cli`.  Then install the `babel-present-env` Babel preset (it will set the plugins/presets necessary for desired target) with the command `npm i -D babel-preset-env`.
```
Comment (rpdasilva) [inline exercises/es6/README.md]: `babel-preset-env` typoe

Comment (ggprod) [inline exercises/es6/README.md]: hehe.. will fix

**Comment by ggprod** [comment]
I seem to have lost the initial branch this PR was started from, but will create another PR to merge the fixes

**Pull request merged**